packages/backend/routes/index.ts

```diff
@@ -0,0 +1,10 @@
+import express, {Request, Response, NextFunction} from 'express';
+
+var router = express.Router();
+
+/* GET home page. */
+router.get('/', function(req: Request, res: Response, next: NextFunction) {
+  res.send('Welcome to Express & TypeScript Server');
```

Comment:
In the production environment, what's gonna be serving your HTML? Are you going to have two servers like on dev (HTML on 3000 and API on 3001) or are you gonna set up a route on your API server to serve the HTML?

Reply:
i was thinking about hosting it as two separate servers. just to void the complications that react-router-dom introduces. But i guess a similar thing could be accomplished by grouping all the backend routes behind `/api` that way the url generated by react-router-dom doesnt get confused with a real API route.